odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -99,6 +99,7 @@ Setup instructions:
     .. note::
 
       It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
+      Any other services running on port:80 need to be stopped as it will prevent the server from running ,for ex: Apache2
```

Comment:
This is helpful content to add! Just a few minor edits. 

"... running," should have comma next to running not "running ," 
please spell out "for ex:" as "for example, ..."